pkg/cluster/config.go

```diff
@@ -32,6 +33,18 @@ const (
 	MachineAPINamespace = "openshift-machine-api"
 )
 
+var (
+	// acceptableProxyVars is a list of the supported proxy variables
+	acceptableProxyVars = []string{"HTTP_PROXY", "HTTPS_PROXY", "NO_PROXY"}
```

Comment:
This is just `proxyEnvVars`. `acceptable` is implicit and need not be called out.